sample-apps/donet-sample-app/donet-sample-app/Controllers/AppController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Amazon.S3;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Net.Http;
+
+namespace donet_sample_app.Controllers
+{
+
+    var meter = new Meter("MyApplication");
```

Comment:
Can a more appropriate application name be chosen?

Reply:
changed it to `donet`